src/views/virtualmachines/tree/components/TreeViewRightClickAction.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { FC } from 'react';
+
+import ActionDropdownItem from '@kubevirt-utils/components/ActionDropdownItem/ActionDropdownItem';
+import useSingleNodeCluster from '@kubevirt-utils/hooks/useSingleNodeCluster';
+import { getName, getNamespace } from '@kubevirt-utils/resources/shared';
+import { Menu, MenuContent, MenuList, Popper, TreeViewDataItem } from '@patternfly/react-core';
+import useVirtualMachineActionsProvider from '@virtualmachines/actions/hooks/useVirtualMachineActionsProvider';
+import { getVMIMFromMapper } from '@virtualmachines/utils/mappers';
+
+import useContextMenuListener from '../hooks/useContextMenuListener';
+import { vmimMapperSignal, vmsSignal } from '../utils/signals';
+
+import { popupbackgroundStyle, treeViewActionMenuStyle } from './constants';
+
+type TreeViewRightClickActionProps = {
+  treeData: TreeViewDataItem[];
+};
+
+const TreeViewRightClickAction: FC<TreeViewRightClickActionProps> = ({ treeData }) => {
+  const [contextMenuTrigger, hide] = useContextMenuListener(treeData);
+  const [isSingleNodeCluster] = useSingleNodeCluster();
+
+  const [namespace, name] = contextMenuTrigger?.id?.split('/') || ['', ''];
+
+  const vmim = getVMIMFromMapper(vmimMapperSignal.value, name, namespace);
+  const vm = vmsSignal?.value?.find(
+    (resource) => getName(resource) === name && getNamespace(resource) === namespace,
+  );
+
+  const [actions] = useVirtualMachineActionsProvider(vm, vmim, isSingleNodeCluster);
+
+  if (!contextMenuTrigger) return null;
+
+  return (
+    <>
+      <Popper
+        popper={
+          <Menu containsFlyout style={treeViewActionMenuStyle}>
```

Comment:
why style and not CN?

Reply:
uh i didn't change it back when I was testing